cwf/cloud/go/services/cwf/servicers/builder_servicer.go

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
I tried to get Git to view this and others as a move, but it only picked up on some of them. I didn't change any of the logic, either in the builders or in the tests, so it should be fine @xjtian